newdle/client/src/components/creation/CreationSuccessPage.js

```diff
@@ -38,7 +38,17 @@ export default function CreationSuccessPage() {
             invite:
           </p>
         )}
-        <div className={styles['newdle-link']}>{newdle.url}</div>
+        <Input
+          className={styles['newdle-link']}
+          disabled
```

Comment:
I think `readOnly` would be more appropriate here, but I'll have to try it out and see it in action. Could even select the contents of the field when focusing it..

Reply:
Yeah, the disabled field is having some problems to select the contents. Let me try with readOnly and I will push accordingly.